### DISCUSSION TITLE
第6节课笔记（20班）

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#473] [笔记]
宝子们，笔记提交到这里

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_6/lesson_6_note.md

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679640002

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679781821

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135878525
